{"name": "test_fixture_scope_module01", "status": "failed", "statusDetails": {"message": "AssertionError: assert 10 < 10", "trace": "ramdom_int_in_module = 10\n\n    def test_fixture_scope_module01(ramdom_int_in_module):\n>       assert ramdom_int_in_module < 10\nE       assert 10 < 10\n\ntest_02_fixture.py:45: AssertionError"}, "start": 1627038694156, "stop": 1627038694156, "uuid": "fbb90914-2ace-4c96-bdca-baa58ac2b29e", "historyId": "a35e67ee532c6b01bb1d805fd977ac3e", "testCaseId": "c7b17272c4c84bd0eb9dd77ecaa46c3f", "fullName": "test_02_fixture#test_fixture_scope_module01", "labels": [{"name": "suite", "value": "test_02_fixture"}, {"name": "host", "value": "JULIZHOU-6JV5P"}, {"name": "thread", "value": "19824-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_02_fixture"}]}